go2_simulation/simple_wrapper.py

```diff
@@ -70,6 +61,9 @@ def __init__(self, model, geom_model, visual_model, q0, args):
         self.v = np.zeros(self.model.nv)
         self.a = np.zeros(self.model.nv)
         self.f_feet = np.zeros(4)
+        self.fext = [pin.Force(np.zeros(6)) for _ in range(self.model.njoints)]
```

Comment:
i dont understand, why do you introduce zeros forces here ? this is not necessary anymore in simple step, please remove